src/editorial/web/components/logo/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { css } from '@emotion/react';
+import { brandText } from '@guardian/src-foundations/palette';
+import { SvgGuardianLogo } from '@guardian/src-brand';
+import { Link } from '@guardian/src-link';
+import { from } from '@guardian/src-foundations/mq';
+import { visuallyHidden } from '@guardian/src-foundations/accessibility';
+
+type LogoType = 'standard' | 'anniversary' | 'bestWebsite';
+
+type Props = { logoType?: LogoType };
+
+const svgSize = css`
+	svg {
+		width: 195px;
+		height: 65px;
+		${from.tablet} {
+			width: 224px;
+			height: 75px;
+		}
+		${from.desktop} {
+			width: 295px;
+			height: 98px;
+		}
+	}
+`;
+
+const svgColour = css`
+	svg {
+		fill: ${brandText.primary};
+	}
+`;
+
+const StandardLogo = () => {
+	return (
+		<div css={[svgSize, svgColour]}>
+			<SvgGuardianLogo />
+		</div>
+	);
+};
+
+const BestWebsiteLogo = () => {
+	return (
+		<div css={svgSize}>
+			<svg
+				fill="none"
+				xmlns="http://www.w3.org/2000/svg"
+				viewBox="0 0 236 92"
```

Comment:
Is there any way we can get the viewbox size consistent with the standard logo ([300x100](https://github.com/guardian/source/blob/2ffa82a5c366b461b8ae18ec9d6fcbe988501d75/src/core/brand/SvgGuardianLogo.tsx#L29))?

Reply:
Yeah, it would be great if these were consistent. @HarryFischer ?

Reply:
This is now done. It did mean that we now no longer use the Source version of the standard logo though because that has a different viewbox. Do you think it's worth aligning everything to the same values as exist in Source or should that logo be updated to match these?